src/pages/ShoppingBasket/component/Product.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import closeImg from '../../../assets/images/cross.png';
+import './Product.scss';
+
+export const Product = ({
+  data,
+  productList,
+  setProductList,
+  checkedState,
+  toggleSelected,
+}) => {
+  const handleCount = value => {
+    if (Number(data.quantity) + value === 0) return;
+
+    const newArray = productList.map(product => {
+      const { productId, quantity } = product;
+      if (productId === data.productId)
+        return { ...product, quantity: quantity + value };
+      return product;
+    });
+    setProductList(newArray);
+  };
+
+  // function onRemove(id) {
+  //   setProductList(productList.filter(item => item.productId !== id));
+
+  function onRemove(id) {
+    fetch('http://10.58.52.215:3000', {
+      method: 'DELETE',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+      body: JSON.stringify(
+        setProductList(productList.filter(item => item.productId !== id))
+      ),
+    }).then(response => response.json());
+  }
+
+  return (
+    <div className="inCartProducts" key={data.productId}>
```

Comment:
```suggestion
    <div className="product" key={data.productId}>
```
여기도 className이 변경 되어야 할 것 같습니다!